database/shift.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Definition des Shift Models
+---
+
+# Shift
+
+Das Shift Model dient zur Repräsentation einer geleisteten "Arbeitsschicht". Eine Schicht ist der Zeitraum zwischen
+dem der User seine Arbeit begonnen hat und diese wieder beendet hat. Es können pro Tag mehrere Schichten erstellt
+werden. Das Modell is t wie folgt definiert :
```

Comment:
```suggestion
werden, allerdings darf sich eine Schicht niemals zwischen zwei Tage fallen. Das Modell ist wie folgt definiert:
```

Reply:
Ich glaube das gehört in die Business Rules.